pipeline-scripts/check-build-promotion.sh

```diff
@@ -0,0 +1,8 @@
+LAST_SUCCESSFUL_BUILD=$(
```

Comment:
Move this script under bin/pipeline-scripts and update paths in .drone.yml accordingly

Reply:
done